src/d3a/models/area/stats.py

```diff
@@ -125,17 +125,14 @@ def get_last_market_stats(self, dso=False):
         out_dict = {}
         if self.current_market is None:
             return out_dict
-        current_market_time_slot_str = self.current_market.time_slot_str
-        out_dict[current_market_time_slot_str] = copy(self.rate_stats_market.get(
-            self.current_market.time_slot, default_trade_stats_dict))
-        out_dict[current_market_time_slot_str]["market_bill"] = \
-            self._get_current_market_bills()
-        out_dict[current_market_time_slot_str]["market_fee_revenue"] = \
-            self._area.current_market.market_fee
+        out_dict = copy(self.rate_stats_market.get(self.current_market.time_slot,
+                                                   default_trade_stats_dict))
+        out_dict["market_bill"] = self._get_current_market_bills()
+        out_dict["market_fee_revenue"] = self._area.current_market.market_fee
         if dso:
-            out_dict[current_market_time_slot_str]["area_throughput"] = \
-                self._get_current_market_area_throughput()
-
+            out_dict["area_throughput"] = self._get_current_market_area_throughput()
+            out_dict["self-sufficiency"] = self.kpi.get("self_sufficiency", None)
+            out_dict["self_consumption"] = self.kpi.get("self_consumption", None)
```

Comment:
sorry just noticed after approving, please maintain uniformity ie either use dash(-) or under-score(_)

Reply:
Great catch! Thanks, will change it